### ISSUE TITLE
Youtube videos open into a hilariously small embed

### ISSUE THREAD

**Issue by jeromechoo** [cheeaun/phanpy#405]
**Describe the bug**
Opening a Youtube video from a post leads to a view with a working video embed but it is extremely small as to be unviewable unless full screened.

**To Reproduce**
1. Go to https://phanpy.social/#/masto.ai/s/111779108628881994
2. Click on the video

**Expected behavior**
Video should scale to something appropriate to the viewport size.

**Screenshots**
Current size:
![CleanShot 2024-01-18 at 15 57 16@2x](https://github.com/cheeaun/phanpy/assets/3942955/5291588b-53bb-4ff9-9983-d7e2c89f997c)

Proposed size:
![CleanShot 2024-01-18 at 16 03 09@2x](https://github.com/cheeaun/phanpy/assets/3942955/b021cf18-ea31-47ce-aac0-58c78cefb64e)

**Desktop (please complete the following information):**
 - OS: 15" Macbook Air M2, 2023
 - Browser: Firefox 
 - Version: 121.0.1 (64-bit)

**Additional context**
Reproducible across latest Chrome and Safari.

**Comment by Alex0007** [comment]
My proposal is: replace link preview with youtube player directly in post body (onClick). This empty modal makes no sense


<img width="625" alt="Screenshot 2024-02-14 at 00 30 52" src="https://github.com/cheeaun/phanpy/assets/1143226/aed75cbe-b2c2-4856-ab44-101415db678f">

**Comment by Alex0007** [comment]
Youtube embeds are still small 

<img width="668" alt="Screenshot 2024-06-13 at 00 00 02" src="https://github.com/cheeaun/phanpy/assets/1143226/c02141c5-ac10-49f6-b8e1-355884012a74">
<img width="668" alt="Screenshot 2024-06-13 at 00 00 14" src="https://github.com/cheeaun/phanpy/assets/1143226/8081b38b-855c-4c86-96f7-2ea67d10bbc3">

**Comment by cheeaun** [comment]
@Alex0007 do you have a link to the post? Suspect the dimensions are from the API response which probably points to the thumbnail sizes. Else, I don't see any way to know the video's actual dimensions (Video is inside an iframe of another site which won't expose any info).

One possible way, is to not follow dimensions from API response and make it as huge as possible, and hopefully the content inside the iframe knows how to fix its own layout 🤔

**Comment by Alex0007** [comment]
@cheeaun yeah it's still hilariously small 😑 i'd say that modal with small youtube frame is useless (blocks every other functionality + video is unwatchable without going to fullscreen because of small size)

**Comment by Alex0007** [comment]
<img width="1624" alt="Screenshot 2024-09-18 at 21 48 46" src="https://github.com/user-attachments/assets/1ff6a809-b892-4675-abfe-de48431ca3da">
Still!
